### PR TITLE
Improve formulation of definition of "valid language tag"

### LINKED ISSUE #1330: "valid" language (sub)tag is not a good term [de46e4, bf051a, 5b7ae0]
The language rules use the term "valid language subtag" from our glossary (https://act-rules.github.io/glossary/#valid-language-subtag).

The word "valid" is not really correct here as the definition doesn't really check that the subtag is valid.

This is made even worse by #1328 which changes it to "valid language tag" but the definition only sets conditions on the primary **sub**tag.

### PR DESCRIPTION
**Pull request by Jym77**
Improve the definition of "valid language tag".
The word "valid" is maybe still a bit confusion (as it doesn't mean "valid according to BCP 47") but that should be OK. 

Closes issue(s):

- closes #1330

Need for Final Call:
This will require a 1 week Final Call (shouldn't change anything, but I'd rather be safe)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: pages/glossary/valid-language-tag.md [hunk 1]
```diff
@@ -5,12 +5,15 @@ unambiguous: true
 objective: true
 ---
 
-A language tag consists of a [primary language subtag][] from the [language subtag registry][], optionally followed by a hyphen (`-`) and any sequence of characters.
+A [language tag][] is _valid_ if its [primary language subtag][] exists in the [language subtag registry][] with a `Type` of `language`.
```
Comment (ajanec01) [inline pages/glossary/valid-language-tag.md:8]: Just wondering if it's worth adding a link to the [Type Field section](https://tools.ietf.org/html/bcp47#section-3.1.3).

**Comment by Jym77** [comment]
@ajanec01 thanks for the comments.

> Minor thing, wouldn't "A language tag is _valid_ if its primary language subtag exists in the language subtag registry **and has** a Type field ..." be easier to read?

I agree it is easier to read, but I think it wouldn't be technically correct 😕 

> A [language tag][] is _valid_ if its [primary language subtag][] exists in the [language subtag registry][] with a [Type field][] whose field-body value is `language`.

The primary language subtag is a string (a substring of the language tag). It doesn't "have" a Type field. The string can be in the IANA registry, and that entry in the registry does have a Type field. So the reference to the Type field must refer to the registry, not to the subtag…

**Comment by WilcoFiers** [comment]
FYI, since Aron doesn't have write access his approval isn't counted by Github. That doesn't matter for ACT TF. This PR is ready for a final call.

**Comment by Jym77** [comment]
Final Call ends on June 25.

**Comment by Jym77** [comment]
Final Call has ended. Merging.

**Pull request merged**